doc/CONFIGURATION.md

```diff
@@ -505,6 +505,32 @@ const node = await Libp2p.create({
 })
 ```
 
+#### Configuring PeerStore
+
+PeerStore persistence is disabled in libp2p by default. You can enable and configure it as follows. Aside from enabled being `false` by default, it will need an implementation of a [datastore](https://github.com/ipfs/interface-datastore).
```

Comment:
It might be good to just add a brief note about not using the memory datastore as that will be ineffective for persistence.